sdk/core/azure-core-experimental/src/main/java/module-info.java

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+module azure.core.experimental {
```

Comment:
Module names should be the same as the root package name `com.azure.core.experimental`.
https://azure.github.io/azure-sdk/java_design.html#modules

Reply:
Fixed